hooks/command

```diff
@@ -187,14 +187,14 @@ else
   do
     set +e
     pod_json=$(kubectl get pod "$pod_name" -o json)
+    set -e
```

Comment:
I don't think this change is needed? this would cause any failure in the commands below to exit the script entirely which wouldn't be what we want?